### PR TITLE
Footer Adjustment

### PR DESCRIPTION
**Pull request by sebworks**
Adjust font-size of label in footer to adhere to design spec.
 
## Changes
 
- Updated 'static/css/footer.less' to adhere to design spec.

## Review
 
- @sebworks 
- @anselmbradford 
 
## Preview
 
screen shot 2015-03-23 at 10 50 09 am

## Notes
 
None

### CODE REVIEW

File: static/css/footer.less [hunk 1]
```diff
@@ -100,9 +100,10 @@
 
         &-label {
             display: block;
+            font-size: unit(16px / @base-font-size-px, em);
             margin-bottom: unit(8px / @base-font-size-px, em);
             .webfont-demi();
-
+
```
Comment (Scotchester) [inline static/css/footer.less]: You accidentally committed some extra whitespace here.

File: static/css/footer.less [hunk 2]
```diff
@@ -100,9 +100,9 @@
 
         &-label {
             display: block;
+            font-size: unit(16px / @base-font-size-px, em);
             margin-bottom: unit(8px / @base-font-size-px, em);
             .webfont-demi();
-
```
Comment (jimmynotjim) [inline static/css/footer.less]: I would say leave this line break since the `respond-to` mixin is similar to nesting.

Comment (Scotchester) [inline static/css/footer.less]: Yes, I didn't mean to imply that the linebreak should be removed.

**Comment by sebworks** [comment]
@Scotchester, sorry for the confusion.

**Comment by jimmynotjim** [comment]
Squash this and then :+1:

File: static/css/footer.less [hunk 3]
```diff
@@ -100,6 +100,7 @@
 
         &-label {
             display: block;
+            font-size: unit(16px / @base-font-size-px, em);
```
Comment (anselmbradford) [inline static/css/footer.less]: [`@base-font-size-px` is `16px`](https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-vars.less#L49), could you say `1em` here? Or would that be problematic if `@base-font-size-px` changed?

**Comment by sebworks** [comment]
That was my concern.  There are a couple places in the file where they are setting the font size so I was following the existing paradigm.

**Comment by Scotchester** [comment]
Are you trying to make this 16px? You'll have to divide by the font size in current context, which is 12px, set on:

```less
@media only all and (min-width: 48em) {
    .footer_general-inqueries {
        font-size: .75em;
    }
}
```

**Comment by sebworks** [comment]
No, I was setting the size to 12px.

**Comment by Scotchester** [comment]
Ah, gotcha. My perspective would be "This label should be the same size as the standard text in this section." So for that reason, I would just set it as @anselmbradford suggested, plain old `font-size: 1em;`. It's less about setting the font size to 12px than it is setting it to the same as its surrounding text. Know what I mean?

**Pull request merged**